test/tcframe/generator/TestCaseGeneratorTests.cpp

```diff
@@ -155,6 +161,22 @@ TEST_F(TestCaseGeneratorTests, Generation_Sample_Successful) {
     EXPECT_THAT(outForInput->str(), Eq("42\n"));
 }
 
+TEST_F(TestCaseGeneratorTests, Generation_Sample_NoOutput_Successful) {
+    {
+        InSequence sequence;
+        EXPECT_CALL(logger, logTestCaseIntroduction(_));
```

Comment:
Argh sorry I was not clear; I mean only use wildcard `_` for the ones with `.Times(0)` only :') please revert the others